gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -11852,6 +11855,7 @@ static PyObject* GemRB_GetSystemVariable(PyObject * /*self*/, PyObject* args)
 		case SV_HEIGHT: value = core->Height; break;
 		case SV_GAMEPATH: strlcpy(path, core->GamePath, _MAX_PATH); break;
 		case SV_TOUCH: value = core->GetVideoDriver()->TouchInputEnabled(); break;
+		case SV_ENTERED_GAME: value = core->EnteredGame; break;
```

Comment:
Wouldn't this work? Then there's no need for another Interface member:
```suggestion
		case SV_ENTERED_GAME: value = core->GetGame(); break;
```

Reply:
As I read the docs of GUIScript, there is always two steps to do: `GemRB.LoadGame`, then doing the CG, then `GemRB.EnterGame`.

`GemRB.LoadGame` is invoked with a null savegame, makes `Interface::SetupLoadGame` set the SG and some flags, so that `Interface::HandleFlags` will call `Interface::LoadGame` and that sets the `game` you proposed there.

I will check later that I didn't miss anything at runtime for that reasoning, right now I just assume that this isn't sufficiently helpful for this issue.

Reply:
Nope, `game` is already set in CG.

Reply:
What about GetSelectedPCSingle? But also Brad is right and we can just fix it for subviews, where it can be cleaner.

Reply:
Ok, then let's agree on the subviews fix.